tasks/discover-pdrs/tests/force.js

```diff
@@ -7,15 +7,15 @@ const fs = require('fs-extra');
 const { RemoteResourceError } = require('@cumulus/common/errors');
 const cloneDeep = require('lodash.clonedeep');
 
-const { discoverPdrs } = require('../index');
-const input = require('./fixtures/input.json');
+const { discoverPdrs } = require('..');
 
 const { recursivelyDeleteS3Bucket, s3, uploadS3Files } = require('@cumulus/common/aws');
 const {
   randomString,
   validateConfig,
   validateOutput
 } = require('@cumulus/common/test-utils');
+const input = require('./fixtures/input.json');
```

Comment:
Worth noting using `require` on json config files is dangerous because of `require` caching the loaded object. If it were modified here, that modified object would be loaded in the index test, which `require`s the same file. This has bitten us in the kinesis integration tests. Better to read in the file or `cloneDeep` it after requiring. While you're changing this, could address that.